src/web/nootils.js

```diff
@@ -73,6 +73,20 @@ module.exports = function(settings) {
     isDateValid: function(date) {
       return !isNaN(date.getTime());
     },
+    getField: function(report, field) {
+      /* field is a string e.g 'dob' (equivalent to report.fields.dob) or
```

Comment:
consider using the jsdoc syntax for this documentation https://devhints.io/jsdoc